types/dot-prop/index.d.ts

```diff
@@ -1,8 +1,17 @@
-// Type definitions for dot-prop
-// Project: https://github.com/sindresorhus/dot-prop
+// Type definitions for dot-prop 4.1
+// Project: https://github.com/sindresorhus/dot-prop#readme
 // Definitions by: Sam Verschueren <https://github.com/samverschueren>
+//                 BendingBender <https://github.com/BendingBender>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
 
+interface DotProp {
```

Comment:
Instead of `export =`-ing an object, you could just use `export function` for each of these methods:
```ts
export function get(...): any;
...
```